### ISSUE TITLE
Bug in `tutorials/speaker_recognition/ASR_with_SpeakerDiarization.ipynb`

### ISSUE THREAD

**Issue by tanujdhiman** [NVIDIA/NeMo#2555]
**Describe the bug**

![e](https://user-images.githubusercontent.com/56601466/127006734-044c514e-b5da-45df-bb8c-480030da3b00.PNG)

**Steps/Code to reproduce bug**

Please list *minimal* steps or code snippet for us to be able to reproduce the bug.

A  helpful guide on on how to craft a minimal bug report  http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports. 

**Expected behavior**

**Environment overview (please complete the following information)**

 - Environment location: [Bare-metal, Docker, Cloud(specify cloud provider - AWS, Azure, GCP, Collab)]
 - Method of NeMo install: [pip install or from source]. Please specify exact commands you used to install.
 - If method of install is [Docker], provide `docker pull` & `docker run` commands used

**Environment details**

If NVIDIA docker image is used you don't need to specify these.
Otherwise, please provide:
- OS version - Google Colab
- PyTorch version 
- Python version - 3.7

**Additional context**

Example: GPU model

**Comment by nithinraok** [comment]
Can you please describe the bug, what audio samples you tried, do you have ground truth labels etc?

**Comment by tanujdhiman** [comment]
I tried a wav file actually https://drive.google.com/file/d/1OquRtjhA6YU6e29VN86-JVeAvi62s_S6/view?usp=sharing

**Comment by nithinraok** [comment]
thanks for sharing the file.
Few things:
* wav file you shared is 8khz -> speaker diarization samples only consider 16Khz samples (use sox to upsample)
* shared wav file has 2 channels -> we do ASR and diarization on only single channel audio files (again use sox to mix 1,2 channels)
* We found a bug while testing on your audio file whose audio length is less than 3 sec, we thought we fixed it, but it wasn't. Now it should be all good with recent fix to r1.2.0 branch. PR: https://github.com/NVIDIA/NeMo/pull/2566 . Thanks
